### PR TITLE
De-duplicate ontology descriptions

### LINKED ISSUE #1965: Some ontologies have duplicate short and long descriptions
The frontmatter can include a `description` field. 16 ontologies in OBO Foundry contain exactly the same text in the `description` field as the long-form markdown that follows the frontmatter. Potentially more contain text that's trivially different. We should do two things:

1. Remove the long-form text that duplicates the short-form text
2. Provide a test to make sure this doesn't happen again

### PR DESCRIPTION
**Pull request by cthoyt**
Closes #1965 by removing the long-form text in 16 ontologies that matches the short form text and institutes a data integrity test to stop this from happening again. The following ontologies are affected:

1. AISM @JCGiron
2. APO @srengel
3. CHEMINF @egonw
4. CTO @akodamullil
5. FBbi @wawong
6. HAO @mjy
7. MAMO @gambardella
8. MPATH @PaulNSchofield
9. OARCS @mjy
10. OHMI @yongqunh
11. OPMI @yongqunh
12. RS @shurjenw
13. SPD @martinjramirez
14. WBbt @raymond91125
15. WBls @chris-grove
16. WBPhenotype @chris-grove

To all of the ontology responsible people, please respond ASAP if there's any reason this change is problematic. Again, note that in each of these 16 ontologies, the text was _exactly_ the same in the `description` field in the structured info and in the long form markdown area. I'd invite all of you to write longer descriptions for your ontologies in separate PRs as well, as many of them were quite short.

### CODE REVIEW

File: ontology/cheminf.md [hunk 1]
```diff
@@ -27,5 +27,3 @@ activity_status: active
 repository: https://github.com/semanticchemistry/semanticchemistry
 preferredPrefix: CHEMINF
 ---
-
-Includes terms for the descriptors commonly used in cheminformatics software applications and the algorithms which generate them.
```
Comment (egonw) [inline ontology/cheminf.md:31]: No objections (info is in struct data in line 16)

Comment (cthoyt) [reply ontology/cheminf.md:31]: yes, this is exactly the case for all ontologies in this PR. Thanks for the comment!

**Comment by matentzn** [comment]
TBH I doubt this requires all these permissions. The information is:

1. outside the metadata record
2. purely for display purposes on the website
3. purely redundant

Happy with a tight timeline and merge on Wednesday 22nd June, 0 am PT.

**Review by matentzn — APPROVED** [review]

**Review by JCGiron — APPROVED** [review]
I have no problem with removing duplictated information from the AISM.

**Comment by raymond91125** [comment]
Yes, makes sense to me. WBbt

**Comment by matentzn** [comment]
This will be merged on Friday, 24th June.

**Comment by cthoyt** [comment]
@matentzn bump :)

**Comment by matentzn** [comment]
Thank you @cthoyt for the bump and the work :) Greatly appreciated.

**Pull request merged**